src/hex/hex_game.rs

```diff
@@ -430,4 +431,54 @@ impl IGame for HexGame {
     fn get_repetition_limit() -> Option<u32> {
         None
     }
+
+    fn produce_transformed_data_entries(entry: DataEntry<Self>) -> Vec<DataEntry<Self>> {
+        let transform = |e: &DataEntry<Self>, transform_sq: &dyn Fn(usize) -> usize| {
+            let (board_red, board_blue) = [e.pos.board_red, e.pos.board_blue]
+                .iter()
+                .map(|b| {
+                    let mut bt = HexBitboard::new();
+                    for idx in 0..HexGame::BOARD_SIZE * HexGame::BOARD_SIZE {
+                        bt.set(transform_sq(idx), b.get(idx));
+                    }
+                    bt
+                })
+                .collect_tuple()
+                .unwrap();
+            let pos = HexPosition::new_from_board(board_red, board_blue, e.pos.get_turn());
+
+            let probs = e
+                .probs
+                .iter()
+                .map(|(m, p)| (HexMove::from_idx(transform_sq(m.to_idx())), *p))
+                .collect_vec();
+
+            let winner = e.winner;
+            DataEntry { pos, probs, winner }
+        };
+
+        let rows_mirror = |e: &DataEntry<Self>| {
```

Comment:
are you sure these are valid transforms in hex? 
i think they x and RowMirror(x) are pretty different positions
the valid transforms are: diagonal (only one of them), 180 degree turn, 180 degree turn + diagonal

Reply:
not sure i followed. why mirroring the rows is not valid? player1 is still trying to connect the left most column to the right most column, and player2 is doing the same for rows.
I think diagonal is valid only if you switch the players, which we dont do because its always player1 turn

Reply:
A:
![image](https://user-images.githubusercontent.com/4618146/200034776-4e4c3d73-e742-4092-96ef-ae9f837e0cdb.png)
RowTransform(A)
![image](https://user-images.githubusercontent.com/4618146/200034839-8d91fa1f-395f-44fe-ab16-17313e103375.png)
Not the same position. Am I missing something?

Reply:
You are correct, i somehow thought that its equivalent to regular grid.
I think 180 rotation is indeed valid
![image](https://user-images.githubusercontent.com/15381794/200113289-f12c7981-ffb5-4ca1-83c4-149999a8d56e.png)
But i think diagonal rotation is valid only if you switch the players, which we can't do, the network always tries to connect top row to bottom one (or columns, dont rememeber which)
![image](https://user-images.githubusercontent.com/15381794/200113330-0b4eaddd-4617-4722-939d-e5d9df502d9d.png)

Reply:
@poja this is settled? only 180 rotation?

Reply:
Yesss settled